### PR TITLE
Textify Medical Nanites

### PR DESCRIPTION
**Pull request by Turtlelord26**
#619 
They now have built-in benefits (minor Shock save bonus added to the already-existing rest healing), and link you to relevant rest and feat rules.
Provided explicit pathway to upgrade from the booster to the full unit.

### CODE REVIEW

File: Items/07_Items.txt [hunk 1]
```diff
@@ -38,12 +38,25 @@ inside it, exportable to a portable device.
...
+may be upgraded to a full unit (see below) at a cost of $700.
+	
+Medical Nanites Unit ($1000):
+	One unit of Medical Nanites. Having at least one unit of Medical Nanites 
+provides all the benefits of a booster shot (see above), with even more potent 
+rest healing. Additionally, full units of medical nanites can be harnessed to 
+execute Active Feats (see Feats in the Basic Rules).
+	Additional units after the first do not provide more healing or save 
+bonus, but may be used to execute more active feats in shorter time frames. The 
+number of Units a character can have at once is related to their level, see the
```
Comment (rosebrallier) [inline Items/07_Items.txt]: We could say "maximum number of Units" to be as super clear as possible.

File: Items/07_Items.txt [hunk 2]
```diff
@@ -38,12 +38,25 @@ inside it, exportable to a portable device.
...
+	Additional boosters after the first have no effect. A booster shot 
+may be upgraded to a full unit (see below) at a cost of $700.
+	
+Medical Nanites Unit ($1000):
+	One unit of Medical Nanites. Having at least one unit of Medical Nanites 
+provides all the benefits of a booster shot (see above), with even more potent 
+rest healing. Additionally, full units of medical nanites can be harnessed to 
+execute Active Feats (see Feats in the Basic Rules).
+	Additional units after the first do not provide more healing or save 
+bonus, but may be used to execute more active feats in shorter time frames. The
```
Comment (rosebrallier) [inline Items/07_Items.txt]: "provide more save bonus" reads really awk to me. Perhaps "do not provide more healing over rests or increase the save bonus"

File: Items/07_Items.txt [hunk 3]
```diff
@@ -38,12 +38,25 @@ inside it, exportable to a portable device.
...
-Medical Nanites Booster Shot ($):
-	Half a unit of Medical Nanites.
+Medical Nanites Booster Shot ($400):
+	A partial unit of Medical Nanites. The booster shot is a standard 
+medical procedure on more affluent worlds, like a vaccine, and provides 
+resistance to and faster healing from disease and injury. Characters with a 
+nanite booster shot gain +10 to Shock saves and benefit from faster healing 
+during Short Rests (see Resting in the Basic Rules).
+	Additional boosters after the first have no effect. A booster shot 
+may be upgraded to a full unit (see below) at a cost of $700.
```
Comment (rosebrallier) [inline Items/07_Items.txt]: Suggestion for the Mostest Clarity: "upgraded to a Medical Nanites Unit"

**Review by rosebrallier — APPROVED** [review]
Yay elaborate the thing!
I left two clarity suggestions and one more critical lang style request.

File: 01_06_Leveling_Up.txt [hunk 4]
```diff
@@ -15,48 +15,48 @@ skill points for that level.
...
-| 16    | "        | 9     |            |            | 13            | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 17    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 18    | "        | 10    |            |            | 14            | 7d10               |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 19    | "        |       |            |            |               | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
-| 20    | "        | 11    | 4          | +1         | 16            | "                  |
-+-------+----------+-------+------------+------------+---------------+--------------------+
```
Comment (bleehu) [inline 01_06_Leveling_Up.txt:62]: That's a reward that is going to feel very good! It's worth calling out that this is going to have major balance ramifications on enemy levels, but I think this is ready to test. Looking forward to it!

File: Items/07_Items.txt [hunk 5]
```diff
@@ -32,18 +32,31 @@ Engineer processes; i.e., a Gunslinger who buys an extruder can't construct
...
+	
+Medical Nanites Unit ($1000):
+	One unit of Medical Nanites. Having at least one unit of Medical Nanites 
+provides all the benefits of a booster shot (see above), with even more potent 
+rest healing. Additionally, full units of medical nanites can be harnessed to 
+execute Active Feats (see Feats in the Basic Rules).
+	Additional units after the first do not provide more healing or increase 
+the save bonus, but may be used to execute more active feats in shorter time 
+frames. The maximum number of Units a character can have at once is related to 
+their level - see the leveling table.
```
Comment (bleehu) [inline Items/07_Items.txt:59]: Blocker. This is a really good idea, but I think we need to add edge-case guard rails. In formal terms, this allows the linear-scale buying of power without depreciating marginal gains. Or in laymen's terms, what happens if someone sells a small spaceship and dumps all of that money into Med Nanites. 
One potential guard rail would be a by-level maximum which is explained diajetically that introducing too many nanites to your system at once is really hard on your immune system. Another guard rail could be a limit to only being able to get one more unit per level.

**Review by bleehu — CHANGES_REQUESTED** [review]
Need a guard rail to prevent out-of-control nanite unit abuse. Once that's done, this will be ready! Great work on the mechanic itself though! Prices all look good, and I think this will be a fabulous reward for players as they level up.

File: Items/07_Items.txt (hunk 5 above)
Comment (Turtlelord26) [reply Items/07_Items.txt:59]: I hadn't considered waiting until level 8 and dumping three units into yourself all at once. I would want to add acquisition rate-limits based on flavor alone.
Are you also asking me to make units after the first cost more?

**Review by bleehu — APPROVED** [review]
Great flavor! I can work with that.

**Pull request merged**